### ISSUE TITLE
Not work all bios

### ISSUE THREAD

**Issue by andrajackboy085** [rafaelvcaetano/melonDS-android#249]
![Screenshot_20210327-124606_melonDS](https://user-images.githubusercontent.com/81408724/112711374-70c72680-8efa-11eb-8d1c-7ef4b470c471.jpg)
* **Device model:** SM-A107F
* **Android version:** Android API 29
* **melonDS version:** Beta 1.3.0

**Problem:**  
Insert a small description of the problem.

**Stack trace:**  
```
me.magnum.melonds.ui.emulator.EmulatorActivity$f: Failed to load ROM: NDS_FAILED
	at me.magnum.melonds.ui.emulator.j$e$a$a.a(Unknown Source:204)
	at f.a.b0.e.e.a.q(Unknown Source:10)
	at f.a.s.a(Unknown Source:14)
	at f.a.b0.e.b.e$a.d(Unknown Source:26)
	at f.a.b0.e.b.l$a.d(Unknown Source:2)
	at f.a.b0.e.b.j$a.run(Unknown Source:21)
	at f.a.r$a.run(Unknown Source:9)
	at f.a.b0.g.k.run(Unknown Source:13)
	at f.a.b0.g.k.call(Unknown Source:0)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)

```

**Configuration:**  
```
RomCrashContext(emulatorConfiguration=EmulatorConfiguration(useCustomBios=true, dsConfigDirectory=//, dsiConfigDirectory=//, internalDirectory=/data/user/0/me.magnum.melonds/files, fastForwardSpeedMultiplier=-1.0, useJit=false, consoleType=DS, soundEnabled=true, micSource=BLOW, firmwareConfiguration=FirmwareConfiguration(nickname=Player, message=Hello!, language=1, favouriteColour=0, birthdayMonth=1, birthdayDay=1), rendererConfiguration=RendererConfiguration(videoFiltering=LINEAR, threadedRendering=true)), romSearchDirUri=content://com.android.providers.downloads.documents/tree/msd%3A109840, romUri=content://com.android.providers.downloads.documents/tree/msd%3A109840/document/msf%3A109847, romPath=/storage/msf/109847, sramPath=/storage/msf/109847.sav)
```

**Comment by andrajackboy085** [comment]
![Screenshot_20210327-124955_melonDS](https://user-images.githubusercontent.com/81408724/112711438-fa76f400-8efa-11eb-9336-35ff50c08774.jpg)
Who load rom?

**Comment by shabbirmahmud** [comment]
Did you placed your rom and bios inside the download folder? Then put your rom outside the download folder and update the directories in emulator settings.It should work.

**Comment by andrajackboy085** [comment]
![Screenshot_20210327-133806_Files](https://user-images.githubusercontent.com/81408724/112712375-b509f500-8f01-11eb-9bb1-1bbe47804fa7.jpg)
Yes download folder bios or rom not work Name DS

**Comment by andrajackboy085** [comment]
![Screenshot_20210327-134041_ZArchiver](https://user-images.githubusercontent.com/81408724/112712436-26e23e80-8f02-11eb-9a15-c3d66753125f.jpg)
Work or not?

**Comment by shabbirmahmud** [comment]
Then move roms and bios outside the download folder and update the directories in emulator settings.The rom should load.

**Comment by rafaelvcaetano** [comment]
If your problem is not yet fixed, try the latest [Beta 1.4.0](https://github.com/rafaelvcaetano/melonDS-android/releases/latest) release. If the problem is not fixed, please open a new issue.
